rpc_jobs/rpc_aio.yml

```diff
@@ -43,10 +46,22 @@
             Prepare Kibana Selenium,
             Kibana Tests,
             Holland,
-            Upgrade,
+            Major Upgrade,
             Cleanup,
             Destroy Slave
           UPGRADE_FROM_REF: "liberty-12.2"
+      - leapupgrade:
+          STAGES: >-
+            Allocate Resources,
+            Connect Slave,
+            Prepare Deployment,
+            Deploy RPC w/ Script,
+            Setup MaaS,
+            Verify MaaS,
```

Comment:
We'd like to start with a minimum system leapfrog, and increase over time. Verify Maas would probably be removed too. We can then add the rest.